docs/content/any/project/changelogs/TEMPLATE.md

```diff
@@ -8,11 +8,11 @@ description: >-
 draft: true
 ---
 
-## `RELEASED_PACKAGE_1` NEW_VERSION
+# `RELEASED_PACKAGE_1` NEW_VERSION
```

Comment:
I had this started at h2 b/c you're only supposed to have a single h1 per page for accessibility & SEO purposes

Reply:
o

Reply:
ok smarter than me